### PR TITLE
IMP print failed query to help debugging

### PR DESCRIPTION
**Pull request by bealdav**
please @legalsylvain

### CODE REVIEW

File: sql_request_abstract/models/sql_request_mixin.py [hunk 1]
```diff
@@ -242,6 +242,8 @@ def _check_execution(self):
             self.env.cr.execute(query)
             res = self._hook_executed_request()
         except ProgrammingError as e:
+            # it helps a lot for debugging to have raw query string without \n
+            print(query)
```
Comment (pedrobaeza) [inline sql_request_abstract/models/sql_request_mixin.py]: Why not emit it as logger.warning or similar?

Comment (bealdav) [reply sql_request_abstract/models/sql_request_mixin.py]: logger'll display `\n` not return to the line when occurs, isn't it ?

Comment (pedrobaeza) [reply sql_request_abstract/models/sql_request_mixin.py]: I don't think so if you pass directly the query, or use `from pprint import pprint; pprint(query)`

Comment (foutoucour) [reply sql_request_abstract/models/sql_request_mixin.py]: You might want to consider `logging.exception` which is a good candidate for this case.
`logging.exception` is a log at error level that embed the previous error raised.
In the following example, I embed a `\n` to double check this case
``` python
In [8]: try:
   ...:     1/0
   ...: except:
   ...:     logger.exception("toto\ntoto")
   ...:
toto
toto
Traceback (most recent call last):
  File "<ipython-input-8-b43f6c8f4df1>", line 2, in <module>
    1/0
ZeroDivisionError: division by zero
```
test done in ipython with python 3.7
what do you think ?

Comment (bealdav) [reply sql_request_abstract/models/sql_request_mixin.py]: I think it's nice. Thank you

**Review by legalsylvain — APPROVED** [review]
LGTM for me. Thanks for the improvment.

@pedrobaeza, @foutoucour, could you update your review ? 

regards.

**Review by pedrobaeza — APPROVED** [review]

**Review by foutoucour — APPROVED** [review]

**Pull request merged**